src/test/java/com/easypost/ErrorTest.java

```diff
@@ -34,4 +55,93 @@ public void testError() throws EasyPostException {
         // so we'll just check fo a generic exception
         assertThrows(Exception.class, () -> Shipment.create(null));
     }
+
```

Comment:
I can't mark the lines because they weren't edited, but now that the VCR bug is corrected, please update the `testError` test to test pulling out each individual piece of an error. You can look to the PHP or Ruby error as an example:

- https://github.com/EasyPost/easypost-ruby/blob/1673ceb6991c223ab8dc18cb1abbf2f9087e4ce1/spec/error_spec.rb

Reply:
resolved